lib/workers/repository/result.ts

```diff
@@ -49,23 +56,30 @@ export function processResult(
   ];
   const enabledStatuses = [CONFIG_SECRETS_EXPOSED, CONFIG_VALIDATION];
   let status: ProcessStatus;
-  let enabled: boolean | undefined;
+  let repoEnabled: boolean | undefined;
```

Comment:
Why rename?

Reply:
I renamed this parameter to make it clearer

Reply:
Try to avoid refactoring combined with feature changing, as it makes regression faults harder to detect and PRs longer to review. Please revert